src/storage/item.go

```diff
@@ -157,11 +180,14 @@ func (s *Storage) ListItems(filter ItemFilter, limit int, newestFirst bool) []It
 	if !newestFirst {
 		order = "date asc, id asc"
 	}
+	if filter.IDs != nil || filter.SinceID != nil || filter.MaxID != nil {
+		order = "i.id asc"
+	}
 
 	query := fmt.Sprintf(`
 		select
 			i.id, i.guid, i.feed_id,
-			i.title, i.link, i.date,
+			i.title, i.link, i.content, i.date,
```

Comment:
`i.content` is deliberately not provided to speed up the web browser load times. the value must be provided for Fever only.

Reply:
fixed by `withContent` argument